Kiosk/App/Networking/ArtsyAPI.swift

```diff
@@ -28,6 +28,7 @@ enum ArtsyAPI {
     
     case MyBiddersForAuction(auctionID: String)
     case MyBidPositionsForAuctionArtwork(auctionID: String, artworkID: String)
+    case MyBidPosition(id: String)
```

Comment:
Ah, I was trying to follow the examples of some of the other endpoints, where the ID refers to the object being requested (e.g. `Artwork`, `Artist`) The ones with more descriptive parameter names seem to be disambiguating multiple ID params. I'm cool either way, but we should probably apply consistently. I'm glad to change them all if you think that would be an improvement.